tutorials/onchain-governance/fast-track-gov-proposal.md

```diff
@@ -0,0 +1,220 @@
+
+By leveraging simulation tools like [Chopsticks](/develop/toolkit/parachains/fork-chains/chopsticks){target=\_blank}, developers and governance participants can:
+
+- Simulate the entire lifecycle of a proposal
+- Test the voting outcomes by varying the support and approval levels
+- Analyze the effects of successfully executed proposal on the network's state
+- Identify and troubleshoot potential issues or unexpected consequences before submitting the proposals
+
+This tutorial will guide you through the process of using Chopsticks to thoroughly test OpenGov proposals, ensuring that when you submit a proposal to the live network, you can do so with confidence in its effects and viability.
+
+## Prerequisites
+
+Before proceeding, ensure the following prerequisites are met:
+
+- **Chopsticks installation** - if you have not installed Chopsticks yet, refer to the [Install Chopsticks](/develop/toolkit/parachains/fork-chains/chopsticks/get-started/#install-chopsticks){target=\_blank} for detailed instructions
+- **Familiarity with key concepts** - you should have a basic understanding of the following:
+    - [Polkadot.js](/develop/toolkit/api-libraries/polkadot-js-api){target=\_blank} 
+    - [OpenGov](/polkadot-protocol/onchain-governance/overview){target=\_blank}
+
+## Setting Up the Project
+
+Before diving into testing OpenGov proposals, you need to set up your development environment. You'll create a TypeScript project and install the necessary dependencies. You'll use Chopsticks to fork the Polkadot network and simulate the proposal lifecycle, while Polkadot.js will be your interface for interacting with the forked network and submitting proposals.
+
+Follow these steps to set up your project:
+
+1. Create a new project directory and navigate into it:
+    ```bash
+    mkdir opengov-chopsticks && cd opengov-chopsticks
+    ```
+
+2. Initialize a new TypeScript project:
+    ```bash
+    npm init -y \
+    && npm install typescript ts-node @types/node --save-dev \
+    && npx tsc --init
+    ```
+
+3. Install the required dependencies:
+    ```bash
+    npm install @polkadot/api @acala-network/chopsticks
+    ```
+
+4. Create a new TypeScript file for your script:
+    ```bash
+    touch test-proposal.ts
+    ```
+
+	!!!note
+		The `test-proposal.ts` file is where you'll write your code to simulate and test OpenGov proposals.
+
+5. Open the `tsconfig.json` file and ensure it includes these compiler options:
+    ```json
+    --8<-- 'code/tutorials/onchain-governance/fast-track-gov-proposal/tsconfig.json'
+    ```
+
+## Submitting and Executing a Proposal Using Chopsticks
+
+It's important to note that you should identify the right track and origin for your proposal. For example, if you're requesting funds from the treasury, select the appropriate treasury track based on the spend limits. For more detailed information, refer to [Polkadot OpenGov Origins](https://wiki.polkadot.network/docs/learn-polkadot-opengov-origins){target=_blank}.
```

Comment:
I think we should redirect to our page about origins, and from there, redirect to the wiki if needed

/polkadot-protocol/onchain-governance/overview/#origins-and-tracks